packages/travix-ui-kit/components/radioButton/radioButton.js

```diff
@@ -45,7 +47,7 @@ function RadioButton(props) {
         readOnly={!onChange}
         type="radio"
       />
-      <label aria-checked={checked} className="ui-radio__label" htmlFor={id} role="radio">
+      <label aria-checked={checked} className="ui-radio__label" htmlFor={id} role="radio" {...labelDataAttributes}>
```

Comment:
Could you move the data attributes spread to the beginning, and put each prop on its own line, similar to the `input` element above it?